src/SFA.DAS.Commitments.Application.UnitTests/Commands/UpdateApprenticeshipStopDate/WhenEditingTheStopDateOfAnApprenticeship.cs

```diff
@@ -209,7 +213,7 @@ public async Task ThenShouldPublishV2PublishApprenticeshipStopDateChangedEvent()
             await Handler.Handle(ExampleValidRequest);
 
             MockV2EventsPublisher.Verify(x =>
-                x.PublishApprenticeshipStopDateChanged(It.IsAny<Commitment>(), It.IsAny<Apprenticeship>()), Times.Once);
+                x.PublishApprenticeshipStopDateChanged(It.Is<Commitment>(p=>p == _testCommitment), It.Is<Apprenticeship>(p=>p == TestApprenticeship)), Times.Once);
```

Comment:
You can pass the variable in without using the delegate if you prefer e.g. 

```
MockV2EventsPublisher.Verify(x => x.PublishApprenticeshipStopDateChanged(_testCommitment, TestApprenticeship), Times.Once);
```

Reply:
totally forgot i could do that thx